phabricator-on-ubuntu/phabricator-install-ubuntu.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Please organize scripts and images according to recommended approach shown here https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#files-folders-and-naming-conventions